src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.Components.Browser" />
+    <Reference Include="Microsoft.AspNetCore.Components.Server.Abstractions" />
```

Comment:
So is this the tradeoff (out of the possible ones) that you ended up going with for this PR? I'm not against it but I think we need to rediscuss after p3.

Reply:
This should not be here, but yeah. Abstractions/Core and we put all things that depend on an HttpContext or similar in there, so that we don't polute Microsoft.AspNetCore.Components (MAC)